ground/gcs/src/plugins/config/cfg_vehicletypes/configccpmwidget.cpp

```diff
@@ -936,33 +938,34 @@ void ConfigCcpmWidget::setMixer()
         &mixerSettingsData.Mixer5Type,
         &mixerSettingsData.Mixer6Type,
         &mixerSettingsData.Mixer7Type,
-        &mixerSettingsData.Mixer8Type
+        &mixerSettingsData.Mixer8Type,
+        &mixerSettingsData.Mixer9Type,
+        &mixerSettingsData.Mixer10Type
     };
 
     //reset all to Disabled
-    for (i=0; i<8; i++)
-        *mixerTypes[i] = 0;
+    for (i = 0; i < 8; i++)
+        *(mixerTypes[i]) = 0;
 
     //go through the user data and update the mixer matrix
-    for (i=0;i<6;i++)
+    for (i = 0; i < 6; i++)
     {
-        if (MixerChannelData[i]>0)
+        if (MixerChannelData[i] > 0 && (uint)MixerChannelData[i] < sizeof(mixerTypes) / sizeof(mixerTypes[0]))
```

Comment:
Not that I found. I'd like to add one but really I'm not a big fan of how this code works so I intend to rewrite it, hence minimal fix here.